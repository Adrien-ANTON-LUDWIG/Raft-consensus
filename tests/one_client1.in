--
LOAD file.txt
$
start 3
@3000
exit