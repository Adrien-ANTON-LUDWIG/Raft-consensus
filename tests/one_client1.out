{"filename":"file.txt","origin":3,"type":12}